app/views/layouts/application.html.erb

```diff
@@ -1,22 +1,31 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>SlobodaTeam</title>
-    <%= csrf_meta_tags %>
+<<<<<<< HEAD
+<head>
+  <title>sloboda-supersonic-taxi</title>
+  <meta charset="utf-8">
+  <base href='/' />
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+  <% if driver_signed_in? %>
+    <%= link_to "Sign out as a driver", destroy_driver_session_path, method: :delete %>
+  <% else %>
+     <%= link_to "Sign in as a driver", new_driver_session_path %> 
+  <% end %>
 
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
-
-	<div align="right">
-		  <% if current_dispatcher.present? %>
+  <div align="right">
+      <% if current_dispatcher.present? %>
         <%= link_to('Sign out', destroy_dispatcher_session_path, :method => :delete) %>
```

Comment:
Please use the Ruby 1.9 hash literal syntax when your hash keys are symbols. 
https://github.com/bbatsov/ruby-style-guide#hash-literals